docs/spec/spec.md

```diff
@@ -74,6 +74,7 @@ The conforming implementation of the specification is released and included in t
         * 4.5.1 [Input Union Types](#451-input-union-types)
         * 4.5.2 [Input Objects](#452-input-objects)
     * 4.6 [Interfaces](#46-interfaces)
+    * 4.7 [Interfaces Implementing Interfaces](#47-interfaces-implementing-interfaces)
```

Comment:
This should be under the `interfaces` section.

Reply:
Fixed with [de567d5](https://github.com/ballerina-platform/module-ballerina-graphql/pull/1084/commits/de567d50b94e6a87ae2b98ebe633641f635e3c8b)